modules/campaignion_petition/campaignion_petition_templates/campaignion_petition_templates.features.uuid_node.inc

```diff
@@ -10,56 +10,6 @@
  */
 function campaignion_petition_templates_uuid_features_default_content() {
   $nodes = array();
-
```

Comment:
This just reverts part of the first commit. Could you squash it into the first commit?

Reply:
Sure. But then it wouldn't point out the underlying issue and that the order is actually on purpose?